src/contexts/ethereum-provider.context.tsx

```diff
@@ -60,9 +73,18 @@ const EthereumProviderProvider: FC = (props) => {
               setProvider(web3Provider);
               setAccount({ status: "successful", data: accounts[0] });
             })
-            .catch(() => {
-              setAccount({ status: "failed", error: "Error loading account" });
-            });
+            .catch((error) =>
+              parseError(error).then((errorMsg) => {
+                if (error instanceof Error && error.message === "User closed modal") {
```

Comment:
Not very proud of this, but how can we otherwise identify a "User closed modal" error from WalletConnect?